pages/common/pr.md

```diff
@@ -0,0 +1,24 @@
+# pr
+
+> Paginate or columnate files for printing.
+> More information: <https://www.gnu.org/software/coreutils/manual/html_node/pr-invocation.html>.
+
+- Print multiple files with a default header and footer:
+
+`pr {{file1 file2 file3}}`
+
+- Print all files in parallel, one in each column without a header or footer:
```

Comment:
parallel is technically the wrong word here. Perhaps better may be "Print all files together, .."